infra/packaging/preset/20220323

```diff
@@ -39,6 +39,7 @@ function preset_configure()
 
   # TODO Use "nncc configure" and "nncc build"
   cmake \
+    -DENABLE_TEST=OFF \
```

Comment:
@mhs4670go , does our one-compiler-test (?) package related with this?

Reply:
There was some discussion.
So, is this PR valid?
For me I'm not sure we can disable for PRESET.

Reply:
https://github.com/Samsung/ONE/blob/f1716af6aa38ef4f8d95f4cb4b899229d050236a/compiler/one-cmds/CMakeLists.txt#L120-L125

I think one-cmds tests won't build with this PR. Removing if condition could be an option because it doesn't have ctest.